internal/security/secretstore/init.go

```diff
@@ -363,35 +363,40 @@ func (b *Bootstrap) BootstrapHandler(ctx context.Context, _ *sync.WaitGroup, _ s
 		os.Exit(1)
 	}
 
-	cert := NewCerts(req, configuration.SecretService.CertPath, rootToken, configuration.SecretService.GetSecretSvcBaseURL(), lc)
-	existing, err := cert.AlreadyinStore()
-	if err != nil {
-		lc.Error(err.Error())
-		os.Exit(1)
-	}
+	// With Vault moving to non-TLS mode, if cert is not present then no need to upload
```

Comment:
Should be more clear in this comment.  The kong token has nothing to do with Vault running in TLS mode or not.
What we are doing is making the kong TLS certificate upload optional as an intermediate step to removing it altogether.